src/middleware/mfa-method-middleware.ts

```diff
@@ -27,3 +30,17 @@ export async function mfaMethodMiddleware(
     next();
   }
 }
+
+const selectMfaMiddleware = (): RequestHandler => {
+  try {
+    const mfaServiceUrl = new URL(getMfaServiceUrl());
+    if (supportMfaPage() && mfaServiceUrl) {
+      return mfaMethodMiddleware;
+    }
+  } catch (e) {
+    logger.error(`selectMfaMiddleware ${JSON.stringify(e)}`);
```

Comment:
Same as the other comment about stringify error

Reply:
Thansk Alfonso, addressed